src/NavBar.tsx

```diff
@@ -149,11 +150,13 @@ const ResponsiveAppBar = () => {
             })}
           </Box>
           <Box>
-            {" "}
-            {savedItem ? (
-              <Typography>Welcome {savedItem} </Typography>
-            ) : (
-              <Typography>Login Here :</Typography>
+            {!!user && (
+              <>
+                {/* uncomment after figuring out size; will show the google profile pic
+                <img src={user.picture} alt="user image" /> 
+                */}
+                {user?.name}
```

Comment:
Show's the user's name in the nav bar. The commented line can show the user's google picture, but it's too big, and I'm too lazy to resize it at the moment.